portal/cookbook-guide.md

```diff
@@ -6,84 +6,257 @@ They are [geoscience](https://en.wikipedia.org/wiki/Earth_science)-focused
-1. Someone from the Pythia team will add you as a member of the ProjectPythia organization
-1. Once you have accepted the invitation, navigate to the settings of your cookbook repository, scroll down to the Danger Zone, and click "Transfer"
-1. Select or type "ProjectPythia", confirm, and transfer
-1. Replace the `repository_url` in the `sphinx/config/html_theme_options` of the `_config.yml` file to point to your cookbook's new GitHub repository within the [ProjectPythia](https://github.com/ProjectPythia) organization
+## Deploying your Cookbook
 
-You will automatically retain write access to your cookbook, but if you would like to add outside collaborators to the repository, contact a member of the Pythia team to be given the Admin role on your cookbook repository.
+Pythia Cookbooks are not just collections of notebooks! They are backed by GitHub-based automation that handles building, testing, previewing, and publishing the books online.
 
-You can open issues, PRs, and continue making edits as necessary.
+### About the publishing pipeline
 
-## Make a Zenodo release of your Cookbook
+Any time you push new content to the `main` branch of your Cookbook repo, actions are triggered that do the following:
+- Create an up-to-date build environment using your `environment.yml` file
+- Execute all the notebooks in that environment
+- Package the book into a styled website
+- Check all the links in the website to make sure they are valid
+- Deploy the styled website so it's visible to the world
 
-1.  On Zenodo toggle on your repository by going to GitHub and then finding your repository. Let a Project Pythia teammember know if you cannot do this.
-1.  On GitHub make a new release! This is on the right nav side of the page from your code-view in the repository. Again ask for help if needed. Note Zenodo badge links will fail until you have made a release.
+### Where is my book?
 
-## Add your cookbook to the Cookbooks Gallery!
+The URL of your published book will depend on where the source repository is housed.
+- If the repo is in the [Project Pythia organization](https://github.com/ProjectPythia), the book is published to `https://projectpythia.org/[YOUR_COOKBOOK_REPO_NAME]/`
+- If the repo is in your personal GitHub space, the book is published to `https://[YOUR_GITHUB_USERNAME].github.io/[YOUR_COOKBOOK_REPO_NAME]/`
+
+Here's a handy trick for finding your published book:
+- On the home page of your GitHub repo, look for the "About" section in the right-hand side bar.
+- Click the "gear" icon next to "About"
+- Select the checkbox "Use your GitHub Pages website".
+- Is that checkbox missing? That likely means you're looking at the "About" section of your _personal fork_, not the upstream fork. Navigate back to `https://github.com/ProjectPythia/[YOUR_COOKBOOK_REPO_NAME]/` and look for it there.
```

Comment:
@clyne I added one last clarification here

Reply:
I should have merged the PR while I had approval, lol. If you have a chance to approve again, that would be terrific.

Reply:
Done!